new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSimpleOperatorValidation.java

```diff
@@ -116,18 +148,70 @@ public void testInstallingOperator() {
         // and here we can set the maximum time we are prepared to wait
         .await().atMost(5, MINUTES)
         // operatorIsRunning() is one of our custom, reusable assertions
-        .until(operatorIsRunning(namespace));
-
-    // Delete service account from unique namespace
-    assertDoesNotThrow(
-        () -> TestActions.deleteServiceAccount(new V1ServiceAccount()
-            .metadata(new V1ObjectMeta().namespace(namespace).name(serviceAccountName))));
-    logger.info("Deleted service account " + serviceAccountName);
-
-    // Delete namespace
-    assertDoesNotThrow(
-        () -> TestActions.deleteNamespace(namespace));
-    logger.info("Deleted namespace: " + namespace);
+        .until(operatorIsRunning(opNamespace));
+
+  }
+
+  @AfterEach
+  public void tearDown() {
+    // uninstall operator release
+    if (opHelmParams != null) {
+      assertThat(uninstallOperator(opHelmParams))
+          .as("Test uninstallOperator returns true")
+          .withFailMessage("uninstallOperator() did not return true")
+          .isTrue();
+    }
+
+    // Delete service account from unique opNamespace
+    if (serviceAccount != null) {
+      assertThatCode(
+          () -> deleteServiceAccount(serviceAccount))
+          .as("Test that deleteServiceAccount doesn not throw an exception")
+          .withFailMessage("deleteServiceAccount() threw an exception")
+          .doesNotThrowAnyException();
+    }
+    // Delete domain namespaces
+    if (domainNamespace1 != null) {
+      assertThatCode(
+          () -> deleteNamespace(domainNamespace1))
+          .as("Test that deleteNamespace doesn not throw an exception")
+          .withFailMessage("deleteNamespace() threw an exception")
+          .doesNotThrowAnyException();
+      logger.info("Deleted namespace: " + domainNamespace1);
+    }
+
+    if (domainNamespace2 != null) {
+      assertThatCode(
+          () -> deleteNamespace(domainNamespace2))
+          .as("Test that deleteNamespace doesn not throw an exception")
+          .withFailMessage("deleteNamespace() threw an exception")
+          .doesNotThrowAnyException();
+      logger.info("Deleted namespace: " + domainNamespace2);
+    }
+
+    // Delete opNamespace
+    if (opNamespace != null) {
+      assertThatCode(
+          () -> deleteNamespace(opNamespace))
+          .as("Test that deleteNamespace doesn not throw an exception")
+          .withFailMessage("deleteNamespace() threw an exception")
+          .doesNotThrowAnyException();
+      logger.info("Deleted namespace: " + opNamespace);
+    }
+
+  }
+
+  private String createUniqueNamespaceInTryBlock() {
```

Comment:
Why the strange name?  It's because you don't want the same name as TestActions.createUninqueNamespace(), correct :(?  The "InTryBlock" portion of the name looks strange.  Can this just be createNamespace()?